hopp/tools/design/wind/floris_helper_tools.py

```diff
@@ -0,0 +1,124 @@
+    return orig_dict
+
+def write_floris_layout_to_file(layout_x,layout_y,output_dir,turbine_desc):
+    """Export wind farm layout to floris-friendly .yaml file.
+
+    Args:
+        layout_x (List[float]): x-coordinates of turbines
+        layout_y (List[float]): y-coordinates of turbines
+        output_dir (str): output folder to write layout file to.
+        turbine_desc (str): turbine name or description.
+    """
+
+    layout_x = [float(x) for x in layout_x]
+    layout_y = [float(y) for y in layout_y]
+
+    layout = {"layout_x":layout_x,"layout_y":layout_y}
+    n_turbs = len(layout_x)
+    output_fpath = os.path.join(output_dir,f"layout_{turbine_desc}_{n_turbs}turbs.yaml")
+    write_yaml(output_fpath,layout)
+
+def write_turbine_to_floris_file(turbine_dict,output_dir):
+    """Export turbine model to floris-friendly .yaml file.
+
+    Args:
+        turbine_dict (dict): turbine entry of floris_config file
+        output_dir (str): output folder to write turbine model file to.
+    """
+    turb_name = turbine_dict["turbine_type"]
+    output_fpath = os.path.join(output_dir,f"floris_turbine_{turb_name}.yaml")
+    new_dict = check_output_formatting(turbine_dict)
+    write_yaml(output_fpath,new_dict)
+
+def check_floris_library_for_turbine(turbine_name):
+    """Check if a turbine exists in the floris internal library.
+
+    Args:
+        turbine_name (str): name of turbine
+
+    Returns:
+        bool: whether turbine exists in floris internal library or not.
+    """
+    floris_library_fpath = INTERNAL_LIBRARY / "{}.yaml".format(turbine_name)
+    if os.path.isfile(floris_library_fpath):
+        return True
+    else:
+        return False
+
+def load_turbine_from_floris_library(turbine_name):
+    """Load turbine model file from floris internal library.
+
+    Args:
+        turbine_name (str): name of turbine
+
+    Raises:
+        FileNotFoundError: if file does not exist in floris internal library.
+
+    Returns:
+        dict: floris turbine model dictionary
+    """
+    floris_library_fpath = INTERNAL_LIBRARY / "{}.yaml".format(turbine_name)
+    if os.path.isfile(floris_library_fpath):
+        turb_dict = load_yaml(floris_library_fpath)
+        turb_dict.pop("power_thrust_data_file")
+        return turb_dict
+    else:
+        raise FileNotFoundError("Floris library file for turbine {} does not exist.".format(turbine_name))
+
+def check_libraries_for_turbine_name_floris(turbine_name,floris_model):
+    is_floris_lib_turbine = check_floris_library_for_turbine(turbine_name)
+    is_turb_lib_turbine = check_turbine_library_for_turbine(turbine_name)
+
+    if is_floris_lib_turbine:
+        turb_dict = load_turbine_from_floris_library(turbine_name)
+        return turb_dict
+    elif is_turb_lib_turbine:
+        floris_model.value("turbine_name",turbine_name)
+        turb_dict = get_floris_turbine_specs(turbine_name,floris_model)
+        return turb_dict
+    else:
```

Comment:
Same here, so the else can be deleted and the rest of the lines can be dedented once.